scripts/classification/imagenet/train_imagenet.py

```diff
@@ -13,18 +12,29 @@
 from gluoncv.model_zoo import get_model
 from gluoncv.utils import makedirs, LRSequential, LRScheduler
 
+import dali
```

Comment:
should we catch the ImportError here?

Reply:
Definitely, I could add catching of this exception. Actually, it could happen only when`train_imagenet.py` was updated, but for some reason, `dali.py` was not downloaded. I will do that.

Reply:
Done.  I added the `ImportError` exception which should happen when `dali-gpu` OR `dali-cpu` is used, but `dali.py` is not in `scripts/classification/imagenet`